File: Original/texttoaudiofile/src/main/java/com/andruid/magic/texttoaudiofile/util/FileUtils.java
package com.andruid.magic.texttoaudiofile.util;

public class FileUtils {
    public static String getFileName(String utteranceId){
        return "news_"+utteranceId+".mp3";
    }

    public static String getUtteranceId(String fileName){
        return fileName.substring(5, fileName.length()-4);
    }
}

File: NewsDailyJava/texttoaudiofile/src/main/java/com/andruid/magic/texttoaudiofile/util/FileUtils.kt
package com.andruid.magic.texttoaudiofile.util

class FileUtils {
    companion object {

        @JvmStatic
        fun getFileName(utteranceId: String): String {
            return "news_${utteranceId}.mp3"
        }

        @JvmStatic
        fun getUtteranceId(fileName: String): String {
            return fileName.substring(5, fileName.length - 4)
        }
    }
}